### PR TITLE
feat: meaningful error message scaleway image not found

### PR DESCRIPTION
**Pull request by abarbare**
##### SUMMARY

This PR introduce a meaningful error message in case the user use a non existing image

```
fatal: [localhost]: FAILED! => {
    "changed": false
}
MSG: Error in getting image d4067cdc-dc9d-4810-8a26-0dae51d7df42 on https://cp-ams1.scaleway.com
```

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

scaleway_compute

### CODE REVIEW

File: test/legacy/roles/scaleway_compute/tasks/main.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # SCW_API_KEY='XXX' SCW_ORG='YYY' ansible-playbook ./test/legacy/scaleway.yml --tags test_scaleway_compute
 
 - include_tasks: state.yml
-- include_tasks: ip.yml
-- include_tasks: security_group.yml
-- include_tasks: pagination.yml
+# - include_tasks: ip.yml
```
Comment (remyleone) [inline test/legacy/roles/scaleway_compute/tasks/main.yml]: remove comments

**Comment by remyleone** [comment]
LGTM after removing comments

File: test/legacy/roles/scaleway_compute/tasks/main.yml (hunk 1 above)
Comment (abarbare) [reply test/legacy/roles/scaleway_compute/tasks/main.yml]: Testing mistake 😄
Thanks for the heads up

**Comment by remyleone** [comment]
LGTM

**Comment by gundalow** [comment]
Merged into `devel` for release in Ansible 2.8

**Pull request merged**